koku/api/ingress/reports/serializers.py

```diff
@@ -37,49 +37,49 @@ class Meta:
             "sources_id",
             "bill_year",
             "bill_month",
+            "schema_name",
```

Comment:
Does a serialized needed to be added for this new field?

Reply:
Correct. It wouldnt pass along the schema data without adding it here.